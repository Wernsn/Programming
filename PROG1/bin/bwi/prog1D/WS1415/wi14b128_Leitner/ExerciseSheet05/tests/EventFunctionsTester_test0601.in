6
title
2000
1
1
1
1
6
title
2000
1
1
5
2
6
title
2000
1
1
5
3
0

