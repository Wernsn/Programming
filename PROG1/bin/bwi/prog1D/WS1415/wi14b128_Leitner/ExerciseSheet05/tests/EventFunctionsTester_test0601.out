compare null: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: -1
0
1
compare null: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: -1
0
1
compare null: enter title: enter year (1582-2199): enter month (1-12): enter day (1-31): enter number of attendees: -1
0
1
